src/main/public/js/select-toggled-content.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function () {
+  $('.timeline-row .form-control-select').on('change', function () {
+
+    var $content = $(this).next()
+
+    if (this.value) {
+      $content
+        .removeClass('js-hidden')
+        .attr('aria-hidden', 'false')
+        .attr('aria-expanded', 'false')
+      $content
```

Comment:
There is some repetition here, looks like text area is cleared on every change, no matter what value is selected

Reply:
done